cli/tools/test.rs

```diff
@@ -1452,3 +1454,56 @@ pub async fn run_tests_with_watch(
 
   Ok(())
 }
+
+/// Creates the stdout and stderr pipes and returns the writers for stdout and stderr.
+pub fn create_stdout_stderr_pipes(
+  sender: UnboundedSender<TestEvent>,
+) -> (std::fs::File, std::fs::File) {
```

Comment:
I need to finish https://github.com/denoland/deno/pull/14299 which would mean that this function will be required in `cli/tools/bench.rs`. I'm thinking it should be moved to `runtime/ops/io.rs` at that point, WDYT?

Reply:
I think it could still remain in the CLI, but moved to a common place that `cli/tools/bench.rs` also uses? `runtime/ops/bench.rs` could mostly be reverted other than `op_print`.

Reply:
> I think it could still remain in the CLI, but moved to a common place that `cli/tools/bench.rs` also uses? `runtime/ops/bench.rs` could mostly be reverted other than `op_print`.

Currently there's no such module. What do you mean by reverted?

Reply:
Oh, I meant cli/ops/bench.rs then.